scripts/core/backfill_learnings.py

```
@@ -0,0 +1,642 @@
+    with ThreadPoolExecutor(max_workers=args.workers) as executor:
+        futures = {
+            executor.submit(
+                _process_one, s, agent_prompt, model, max_turns, timeout
+            ): s
+            for s in claimed_batch
+        }
+        for i, future in enumerate(as_completed(futures), 1):
+            s = futures[future]
+            try:
+                result = future.result()
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)